web/app/scripts/export/export-partial.html

```diff
@@ -0,0 +1,34 @@
+<div class="tool export" ng-class="{ 'open': ctl.isOpen }">
+    <div class="tool-tab" ng-click="ctl.toggle()" >
+        <span class="glyphicon glyphicon-share"
+              ng-if="ctl.isOpen || (!ctl.pending &amp;&amp; !ctl.downloadURL)"></span>
```

Comment:
Yeah, my editor makes them white on glaring red.  It's quite a persuasive highlighting scheme.  I should figure out how to add an exception.